src/sphinx/general/assertions.rst

```diff
@@ -115,6 +115,10 @@ Conditions can be chained to apply several conditions on the same metric.
 
 * ``around(value, plusOrMinus, inclusive = false)``: same as above but doesn't include bounds
 
+* ``deviatesAround(target, percentDeviationThreshold)``: check that metric is around a target value plus or minus a given relative margin
+
+* ``percentDeviationThreshold(target, percentDeviationThreshold, inclusive = false)``: same as above but doesn't include bounds
```

Comment:
deviatesAround, not percentDeviationThreshold

Reply:
facepalm....